.golangci.yml

```diff
@@ -5,7 +5,7 @@ linters:
   enable:
     - gofmt
     - govet
-    - depguard
+    #- depguard # there's a bug in depguard with (or just exacerbated by?) Go 1.19 that causes it to have false positives
```

Comment:
Just out of curiosity do we know if this is the case? and if it's been filed against depguard?

Reply:
didn't see @LanceEa comment in https://github.com/emissary-ingress/emissary/pull/4308#discussion_r968606292 my bad